build.gradle

```diff
@@ -51,52 +46,102 @@ allprojects {
     }
 }
 
-bintrayConfig {
-    user = System.getenv('CORDA_BINTRAY_USER')
-    key = System.getenv('CORDA_BINTRAY_KEY')
-    repo = 'corda'
-    org = 'r3'
-    licenses = ['Apache-2.0']
-    vcsUrl = 'https://github.com/corda/corda'
-    projectUrl = 'https://github.com/corda/corda'
-    gpgSign = true
-    gpgPassphrase = System.getenv('CORDA_BINTRAY_GPG_PASSPHRASE')
-    publications = ['cordformation', 'quasar-utils', 'cordform-common', 'api-scanner', 'cordapp']
-    license {
-        name = 'Apache-2.0'
-        url = 'https://www.apache.org/licenses/LICENSE-2.0'
-        distribution = 'repo'
+def publishProjects = project.subprojects - project('api-scanner:annotations')
+
+configure(publishProjects) { subproject ->
+    apply plugin: 'java'
```

Comment:
I'm slightly concerned that you're always applying the `java` plugin, even for plugins written in Kotlin or Groovy, but I suppose it makes no material difference to how everything is compiled.

Reply:
Yeah, we do it in the main corda repo too. Maybe we can fine grain it later but this was needed to get the sourceJar and docsJar tasks working.